docs/src/main/tut/datatypes/freemonad.md

```diff
@@ -560,6 +560,67 @@ val initialState = Nil
 val (stored, _) = state.run(initialState).value
 ```
 
+Another example is more basic usage of `FreeT` with some context `Ctx` for which we provide `Future` interpreter,
+combined with `OptionT` for reducing boilerplate.
+
+```tut:book
+import cats.free._
+import cats._
+import cats.data._
+import cats.implicits._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+sealed trait Ctx[A]
+
+case class Action(value: Int) extends Ctx[Int]
+
+def op1: FreeT[Ctx, Option, Int] =
+  FreeT.liftF[Ctx, Option, Int](Action(7))
+
+def op2: FreeT[Ctx, Option, Int] =
+  FreeT.liftT[Ctx, Option, Int](Some(4))
+
+def op3: FreeT[Ctx, Option, Int] =
+  FreeT.pure[Ctx, Option, Int](1)
+
+val opComplete: FreeT[Ctx, Option, Int] =
+  for {
+    a <- op1
+    b <- op2
+    c <- op3
+  } yield a + b + c
+
+
+/* Our interpreters e.g. for Future, DBIO, Id  */
```

Comment:
I'd say let's not use `Future` in our documentation. Just don't want to give people the impression that it's not a bad idea to use `Future`.  For the purpose of your example, we could use `Either` or something

Reply:
I ended up with `Try` instead of `Future`.

You have mentioned about adding FAQ item about moving away from `Future`.
So e.g. short paragraph about crucial reasons providing some sources for further reading?
Something like [cats effect docs](https://typelevel.org/cats-effect/datatypes/io.html#on-referential-transparency-and-lazy-evaluation) comparison?

Reply:
One of the jobs has failed somehow with "GC overhead limit exceeded" and I'm not able to rerun it. Could you rerun it? Or should I trigger it (e.g. close and open pull)?

Reply:
Sorry about the last response. Yeah, that cats effect docs is great as a further reading!